components/Header/Header.jsx

```diff
@@ -12,17 +12,39 @@ const HeaderContainer = styled.header`
   display: flex;
   align-items: center;
   justify-content: space-between;
-  padding: 0 1rem;
+  padding: 0 2rem;
   background-color: white;
+  box-shadow: 0 5px 10px rgba(0, 0, 0, 0.08);
   z-index: 9999;
 `;
 
+const LogButton = styled.button`
+  background-color: #1fab89;
+  color: black;
+  border: none;
+  border-radius: 4px;
+  padding: 0.5rem 1rem;
+  font-weight: 500;
+  cursor: pointer;
+  transition: all 0.2s ease;
+
```

Comment:
```suggestion
  width: 100px;
```

Maybe you could add a fixed width so that the words "log in" don't break?

Reply:
Amazing! Thanks for the suggestion :)